examples/specs/brush.vl.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "Drag out a rectangular brush to highlight points.",
+  "data": {"url": "data/cars.json"},
+  "selection": {
+    "brush": {
+      "type": "interval"
+    }
+  },
+  "mark": "point",
+  "encoding": {
+    "x": {"field": "Horsepower", "type": "quantitative"},
+    "y": {"field": "Miles_per_Gallon", "type": "quantitative"},
+    "color": {
+      "field": "Cylinders", "type": "ordinal",
+      "condition": {
```

Comment:
@arvind  A follow up question

Stylistically, I'm wondering if we should put condition before other properties so it feels more like if-else
```
condition: { value for condition }
...value for else...
```